doc/intern.rst

```diff
@@ -17,23 +17,29 @@ Directory structure
 
 The |NimSkull| project's directory structure is:
 
-============   ===================================================
-Path           Purpose
-============   ===================================================
-`bin`          generated binary files
-`build`        generated C code for the installation
-`compiler`     the |NimSkull| compiler itself; note that this
-               code has been translated from a bootstrapping
-               version written in Pascal, so the code is **not**
-               a poster child of good |NimSkull| code
-`config`       configuration files for |NimSkull|
-`dist`         additional packages for the distribution
-`doc`          the documentation; it is a bunch of
-               reStructuredText files
-`lib`          the |NimSkull| library
-`tests`        the tests for |NimSkull|, including the language
-               specification under `tests/lang`
-============   ===================================================
+- ``bin/``, ``build/`` - these directories are empty, but are used when Nim
```

Comment:
Duplicate explanation in the readme and intern.rst, should only be contained in the `intern.rst`, and readme one was better formatted, so I moved it here.

Reply:
I kinda wish we just had markdown, or one source of truth, because I'm pretty sure we want to excerpt and not just link information.